clients/unshaded/src/main/java/tachyon/hadoop/HdfsFileInputStream.java

```diff
@@ -45,6 +45,7 @@
   private Path mHdfsPath;
   private Configuration mHadoopConf;
   private int mHadoopBufferSize;
+  private FileSystem.Statistics mStatistics;
```

Comment:
Why do we need to specify `FileSystem.Statistics` instead of just `Statistics`? Do we have something else called Statistics already?